packages/core-components/src/components/sections/IntroPanelImage.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+type IntroPanelImageProps = {
+  image: string;
+};
+
+const IntroPanelImage: React.FC<IntroPanelImageProps> = ({
+  image
+}) => {
+  return (
+    <div>
+      <img 
+        className='w-full h-[400px]'
+        src={image}
+      />
+    </div>
+  );
+};
```

Comment:
```suggestion
}) => (
    <div>
      <img 
        className='w-full h-[400px]'
        src={image}
      />
    </div>
  );
```


*small, can resolve in later PR